### DISCUSSION TITLE
Unable to run Vertipaq Analyzer

### DISCUSSION THREAD

**Discussion by edhans** [TabularEditor/TabularEditor3#1138] [🙋♀️TE3 Q&A] [answered]
**Describe the issue**
Won't run. Shows this error message.
![image](https://user-images.githubusercontent.com/45298358/236954831-c0a64504-9d07-4226-b71b-8adad3e1e1f2.png)

** Which version are you currently using? **
3.6 Enterprise against Power BI Premium capacity.

DAX Studio 3.0.7 does it just fine.

It is a field in a field parameter. Is this a known issue or should I send a BIM file to support?

**Comment by otykier** [comment, marked as answer]
Hi Ed - could you try to enable the **Read statistics from data**-option under **Tools > Preferences > VertiPaq Analyzer** (and if it's already enabled, try to disable it?)

![image](https://github.com/TabularEditor/TabularEditor3/assets/8976200/d7dac2fa-8228-4bf5-94a6-b3ac72e15df1)

**Comment by edhans** [comment]
It was enabled. Disabling it allowed it to successfully run. I think I've turned that on/off many times in the last year or so with issues. Seems random as to when it works and when it doesn't. 

What are the disadvantages to it being off other than the sample violations not returned in Relationships?

**Comment by otykier** [comment]
Not sure - maybe @marcosqlbi can chime in on why VertiPaq Analyzer has this option?

Anyway, we were a couple of versions behind with TE3. Our next release, due next week, will include the latest version of VertiPaq Analyzer, which fixes some issues - for example, the version we use now does not support Dynamic Format Strings, but this has been fixed.
